fetch_rapids.cmake

```diff
@@ -1,5 +1,5 @@
 # =============================================================================
-# Copyright (c) 2018-2022, NVIDIA CORPORATION.
+# Copyright (c) 2018-2023, NVIDIA CORPORATION.
```

Comment:
Seems like pre-commit got confused again. I'd like to improve the hook at some point. For now this can be manually undone.